consensus/src/chained_bft/consensus_types/block.rs

```diff
@@ -274,10 +274,31 @@ where
         &self.payload
     }
 
-    pub fn verify(
+    /// Verifies that the proposal and the QC are correctly signed.
+    /// If this is the genesis block, we skip these checks.
+    pub fn validate_signatures(
         &self,
         validator: &ValidatorVerifier,
     ) -> ::std::result::Result<(), BlockVerificationError> {
+        // if genesis block, we don't verify anything
+        if self.is_genesis_block() {
+            return Ok(());
+        }
+        // verify signature from leader if it's a real proposal
```

Comment:
This comment is confusing to me. In fact I don't think we need any comment here at all :)

Reply:
I argued this elsewhere, I think it's clearer at a glance with comments and facilitate auditing. But in the interest of landing this PR I can remove it. Did you mean all comments in this function or just this one?

Reply:
No strong opinions here :). I'm fine if you land it as is, would probably be happier if you'd remove this comment.